### ISSUE TITLE
[Bug] [Twitch] International name @ mention event doesn't trigger

### ISSUE THREAD

**Issue by dbCoMPy** [SevenTV/Extension#471]
I have an international name set and if someone @s me using the international name, it doesn't trigger the mention event, it just shows the message normally. If I get an @ using my english twitch name, everything works as expected.

**Comment by Batyodie** [comment]
@dbCoMPy Hi. Can you please give an example of your international name.

**Comment by dbCoMPy** [comment]
@Batyodie Yes my international name is Japanese, it's ちびコンピ. I hope that helps.
